driver/ppm_fillers.c

```diff
@@ -3867,6 +3875,48 @@ int f_sys_preadv64_e(struct event_filler_arguments *args)
 }
 #endif /* CAPTURE_64BIT_ARGS_SINGLE_REGISTER */
 
+int f_sys_readv_e(struct event_filler_arguments *args)
+{
+	unsigned long val;
+	int32_t fd;
+	int res;
+
+	/*
+	 * fd
+	 */
+	syscall_get_arguments_deprecated(current, args->regs, 0, 1, &val);
+	fd = (int32_t)val;
+	res = val_to_ring(args, (int64_t)fd, 0, false, 0);
+	CHECK_RES(res);
+
+	return add_sentinel(args);
+}
+
+int f_sys_preadv_e(struct event_filler_arguments *args)
```

Comment:
Same as BPF could we reuse the same filler `f_sys_preadv64_e` as you have done with `f_sys_pread64_e`?